CONVENTIONS.md

```diff
@@ -120,6 +120,21 @@ front of the coordinate (`velocityX`, not `xVelocity`).
 
 Name screen coordinates `x` and `y`, rather than `left` and `top`.
 
+### Network Packets
+
+Always suffix packet names with `S2CPacket` (server to client) or `C2SPacket` (client to server).
+
+Packet names should be in the form of `NounVerbed`: `ResourcePackSentS2CPacket`, `BlockUpdatedS2CPacket`, `PlayerMovedC2SPacket`, etc.
+In some cases, the suffix `WithX` can come after the verb. This is to prevent situations where an awkward `And` must be inserted, such as `PlayerAndEntityInteractedC2SPacket`, over `PlayerInteractedWithEntityC2SPacket`.
```

Comment:
```suggestion
In some cases, the suffix `WithX` can come after the verb. This is to prevent situations where an awkward `And` would be inserted, avoiding `PlayerAndEntityInteractedC2SPacket`, in favor of `PlayerInteractedWithEntityC2SPacket`.
```